session1/challenge/client/src/js/components/presentational/ModalPortal.js

```diff
@@ -13,7 +13,16 @@ const Modal = ({ open, onClick, children }) => {
   );
 }
 
-const Portal = null;
+const Portal = ({ open, onClick, children }) => {
+  return (
+    createPortal(
+      <Modal open={open} onClick={onClick}>
+        {children}
+      </Modal>,
+      document.getElementsByTagName('body')[0]
```

Comment:
We can also use `document.body`.

Reply:
yes, you're right, the "document.body" is better